src/3D/Camera.h

```diff
@@ -30,8 +34,12 @@ class Camera
 	    , _dv(0.0f, 0.0f, 0.0f)
 	    , _projectionMatrix(1.0f)
 	    , _velocity(0.0f, 0.0f, 0.0f)
-	    , _movementSpeed(0.0005f)
+	    , _movementSpeed(0.0000f)
+	    , _maxMovementSpeed(0.0005f)
 	    , _freeLookSensitivity(1.0f)
+	    , _focused(false)
+	    , _focusedPosition(0.0f)
+	    , _focusedPositionRadius(0.0f)
 	{
 	}
 	Camera()
```

Comment:
You should move all function definitions that aren't very basic to an implementation file.
It'll speed up compilation and make it more clean to read and maintain.

Reply:
Removed camera focusing until temple movement is better understood